BDNS_Governance_model.md

```diff
@@ -225,6 +231,16 @@ Anyone can submit a new name to the register. The process for submission is as f
     2. Alternative submission mechanisms (e.g. through a mail address) may be made available if the Github PR mechanisms are deemed too technical.
 2. A notification of the submission is made available to the Community Group
 3. If there are no objections from members of the Community Group or Editors, the editors will either integrate the submission or suggest an alternative to the submitter within two weeks.
-4. If there are objections from members of the Community Group or Editors, the issue is escalated to the attention of the Community Group and its Chair(s), and the normal consensus building process applies
+4. If there are objections from members of the Community Group or Editors, the issue is escalated to the attention of the Community Group and its Chair(s), and the normal consensus building process applies.
 
 Any change to the register of names will follow the process for [proposing specification changes](#heading=h.a8exiljj8ajq) and the [consensus building rules](#heading=h.xhh6rjqudxhq).
+
+
+## Funding
```

Comment:
@richardkreid @KenEnright Funding section added, please review and confirm it is OK.

Reply:
should we say 'awareness' or 'transparency' instead?

Reply:
@RitaLav you mean instead of 'funding'?

Reply:
@pisuke I think we should add 'FPC and Buildings IoT' as well as Arup in the sentence "arup worked in 2021..." Also think the year should be 2020-2021.

Did you resolve the conversation with Oli on the call?

Reply:
I've added also FPC and Buildings IoT in the funding note.